packages/select/src/vaadin-select-item.js

```diff
@@ -3,18 +3,51 @@
  * Copyright (c) 2017 - 2023 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { Item } from '@vaadin/item/src/vaadin-item.js';
+import { html, PolymerElement } from '@polymer/polymer/polymer-element.js';
+import { DirMixin } from '@vaadin/component-base/src/dir-mixin.js';
+import { ItemMixin } from '@vaadin/item/src/vaadin-item-mixin.js';
+import { ThemableMixin } from '@vaadin/vaadin-themable-mixin/vaadin-themable-mixin.js';
 
 /**
  * An element used internally by `<vaadin-select>`. Not intended to be used separately.
  *
- * @extends Item
+ * @extends HTMLElement
+ * @mixes DirMixin
+ * @mixes ItemMixin
+ * @mixes ThemableMixin
  * @protected
  */
-class SelectItem extends Item {
+class SelectItem extends ItemMixin(ThemableMixin(DirMixin(PolymerElement))) {
```

Comment:
The items don't look right:

https://user-images.githubusercontent.com/1222264/214258224-964c8cfc-f4a3-4b01-a23f-14518d444f5f.mp4

Reply:
This could be related to the fact that `vaadin-select` no longer defines `vaadin-item` custom element.
Updated the dev page and visual tests to use `items` API which we probably should recommend more.